src/callbacks_step/analysis.jl

```diff
@@ -496,6 +496,8 @@ pretty_form_ascii(::Val{:l2_divb}) = "l2_divb"
 pretty_form_utf(::Val{:linf_divb}) = "L∞ ∇⋅B"
 pretty_form_ascii(::Val{:linf_divb}) = "linf_divb"
 
+pretty_form_utf(::typeof(lake_at_rest_error)) = "∑(H₀-(h+b))"
```

Comment:
I thought because this function is used as an `extra_analysis_integrals` (e.g. in `elixir_shallowwater_dirichlet.jl`) that this value is summed over the whole domain. Am I mistaken?

Reply:
But you will probably sum the absolute value of it, don't you?

Reply:
yes, you are right. I am summing the absolute value so this should be changed

Reply:
Done.